tests/evergreen/data/distro.json

```diff
@@ -0,0 +1,79 @@
+{
+    "name": "amazon1-2018-build",
+    "user_spawn_allowed": true,
+    "provider": "ec2-auto",
```

Comment:
the provider determines (to some extent) the distro settings.... which might be an issue at some point?

Reply:
I might stop by and ask you about this today.